cdk/cdk-resources/encryption-at-rest/.github/workflows/build.yml

```diff
@@ -104,88 +104,3 @@ jobs:
         run: cd .repo && npx projen package:js
       - name: Collect js Artifact
         run: mv .repo/dist dist
-  package-java:
```

Comment:
[q] Why are there so many changes in the build config here? Wondering are these files shouldn't be committed, or a general cleanup?

Reply:
Yes I am still gaining clarity on exactly what files should be committed and have asked @SuperMohit to confirm on this PR.

Reply:
We can remove these files. Everything can be recreated using projenrc.

Reply:
@SuperMohit So then as you mentioned before, the commit should include only index.ts and projenrc right? Are we good for this PR or should I revert everything else?

Reply:
Discussed offline with @SuperMohit. Keeping files in PR as is for now since for current structure files are kind of specific to each resource.